torch/ao/quantization/fx/_model_report/model_report_observer.py

```diff
@@ -0,0 +1,77 @@
+import torch
+from torch.ao.quantization.observer import ObserverBase
+
+
+class ModelReportObserver(ObserverBase):
+    r"""This observer is used to record additional information regarding keeping track
+    of S = average_batch_activation_range/epoch_activation_range.
+
+    The purpose of this information is to prepare a report to present to users on whether
+    Dynamic or Static Quantization is more appropriate for their model given the general
+    distributions of their data.
+
```

Comment:
please add some docs for attributes as well, e.g. https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fake_quantize.py#L89